client/src/ui/organisms/header/index.scss

```diff
@@ -102,7 +102,7 @@
     }
 
     .auth-container {
-      align-self: center;
+      // align-self: center;
```

Comment:
left-over debugging?

Reply:
Ya, this is what was causing the lowered alignment for the "Sign in" button. I will remove it with my next push.